src/Servers/Kestrel/Core/ref/Microsoft.AspNetCore.Server.Kestrel.Core.netcoreapp.cs

```diff
@@ -204,14 +204,6 @@ public enum HttpMethod : byte
         Custom = (byte)9,
         None = (byte)255,
     }
-    public partial class HttpParser<TRequestHandler> : Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.IHttpParser<TRequestHandler> where TRequestHandler : Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.IHttpHeadersHandler, Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.IHttpRequestLineHandler
```

Comment:
Removing these I think breaks benchmark apps @sebastienros to confirm

Reply:
I think you may need to update the benchmark apps with new namespaces.

Reply:
Actually I'm not sure at all. I don't see any of these calls in the benchmark repo.